### PR TITLE
Revelation helpdesk connector certification

### PR DESCRIPTION
**Pull request by MVliegenthart**
---
#### Important: Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -360,4 +360,5 @@ build/
 dist/
```
Comment (sriyen-msft) [inline .gitignore:360]: @MVliegenthart Please remove the files which are not related to your connector.

File: certified-connectors/Revelation helpdesk/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+﻿
+## Revelation Helpdesk Connector
+The Revelation helpdesk connector allows you to create and update items in your helpdesk such as tickets, clients, users and assets by connecting to the Revelation helpdesk API using OAuth authentication. You can also take advantage of the extensive list of triggers allowing you to integrate your business processes based on events that occur in Revelation helpdesk.
+
+
```
Comment (sriyen-msft) [inline certified-connectors/Revelation helpdesk/README.md:5]: @MVliegenthart Please refer the template provided readme.md file and add required sections.

File: certified-connectors/Revelation helpdesk/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          "displayName": "Revelation URL",
+          "description": "Your Revelation site URL (Example: https://support.yourdomain.com)",
+          "tooltip": "Provide your Revelation helpdesk site URL. This must be a secure Url (https://) and must NOT end with '/'"
+        }
+      },
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Revelation.PowerAutomate",
```
Comment (sriyen-msft) [inline certified-connectors/Revelation helpdesk/apiProperties.json:19]: @MVliegenthart Can you please replace it with `"[[DUMMY]]"`

File: certified-connectors/Revelation helpdesk/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Login with your Revelation helpdesk credentials",
+          "displayName": "Login with your credentials",
+          "tooltip": "Login with your credentials"
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
Comment (sriyen-msft) [inline certified-connectors/Revelation helpdesk/apiProperties.json:51]: @MVliegenthart You have agreed not to have this iconBrandColor for your connector. 

image

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Revelation helpdesk/README.md (hunk 2 above)
Comment (MVliegenthart) [reply certified-connectors/Revelation helpdesk/README.md:7]: @sriyen-msft I have added the additional required sections.

File: .gitignore (hunk 1 above)
Comment (MVliegenthart) [reply .gitignore]: @sriyen-msft I think I've managed to remove the .gitignore file (sorry I'm quite new to Github).

File: certified-connectors/Revelation helpdesk/apiProperties.json (hunk 3 above)
Comment (MVliegenthart) [reply certified-connectors/Revelation helpdesk/apiProperties.json]: @sriyen-msft this is done.

File: certified-connectors/Revelation helpdesk/apiProperties.json (hunk 4 above)
Comment (MVliegenthart) [reply certified-connectors/Revelation helpdesk/apiProperties.json]: @sriyen-msft Apologies for the oversight, this has been fixed.

**Review by sriyen-msft — APPROVED** [review]
